lib/logstash/filters/array_of_maps_value_update.rb

```diff
@@ -0,0 +1,44 @@
+# encoding: utf-8
+
+module LogStash module Filters
+  class ArrayOfMapsValueUpdate
+    def initialize(iterate_on, field, destination, fallback, lookup)
+      @iterate_on = ensure_reference_format(iterate_on)
+      @field = ensure_reference_format(field)
+      @destination = ensure_reference_format(destination)
+      @fallback = fallback
+      @use_fallback = !fallback.nil?
```

Comment:
trick: can also be written as `@use_fallback = !!fallback`

Reply:
I have used that trick before. In this case, fallback being `false` is legitimate.

Reply:
ok! (but its a bit counter intuitive when reading that code that `@use_fallback` would be `true` when `fallback` is false)

Reply:
In this case `@use_fallback` is true when the user supplied a `fallback` value in the config, i.e. it is not nil.
I'll add a comment.